### ISSUE TITLE
[capacity-filter] shows nodes as offline when they are not (e2e support)

### ISSUE THREAD

**Issue by muhamadazmy** [threefoldtech/grid_weblets#1333]
### Description
[devnet]

After following the proper steps to find a node to deploy a VM

![image](https://user-images.githubusercontent.com/10920323/221535375-b8646799-8e23-49aa-b002-06d561c22ecf.png)

It always shows whatever node I select as offline. I checked the dashboard and this node was offline by the time i was chekcing

### Steps to reproduce
- Start deploying a vm
- Select Capacity Filter
- Select Freefarm
- Select Belgium
- Select any node in the list

**Comment by AhmedHanafy725** [comment]
I think this happens as the twin has a public key set, no?
if yes, the end-to-end encryption is still in progress in the ts client https://github.com/threefoldtech/rmb-sdk-ts/issues/17

**Comment by ramezsaeed** [comment]
@muhamadazmy

**Comment by xmonader** [comment]
Now that we are going to disable that E2E feature for a bit it should be fine

**Comment by xmonader** [comment]
blocked on https://github.com/threefoldtech/rmb-sdk-ts/pull/20

**Comment by amiraabouhadid** [comment]
should be unblocked now, e2e is implemented in rmb-sdk-ts now

**Comment by mohamedamer453** [comment]
The issue is still present on devnet weblets https://github.com/threefoldtech/grid_weblets/commit/cc76dc9aa99c24b2efab0b65cc2be73ea79f2458 i tried creating a new twin on devnet and after creating it i checked the pk from the gridproxy and it was set to `0x` 

![image](https://user-images.githubusercontent.com/101194226/226168020-84d59a26-bf93-4c4d-9543-4f46672b62fe.png)

and from the weblets' side the nodes still appear as offline for my newly created twin 

![image](https://user-images.githubusercontent.com/101194226/226168122-76a9d6b1-093c-4bb0-8332-69e27e58cc25.png)

and to confirm this i tried running the [`update_twin.ts` ](https://github.com/threefoldtech/grid3_client_ts/blob/development/scripts/update_twin.ts)script from tsclient and the pk was updated but it also didn't work and the nodes were offline.

![image](https://user-images.githubusercontent.com/101194226/226168375-bd8d363d-b72d-4ca9-8945-46944338cac6.png)

and when i tried to deploy from tsclient i got this error

```
Error: Error getting node 11: Error: Failed to send request to twinId 21 with command: zos.statistics.get, payload:  due to 401 e2e encryption error: k256: malformed public key
```

**Comment by A-Harby** [comment]
Tried with a twin that have a pk.
![image](https://user-images.githubusercontent.com/109473873/226894509-8d163dd9-86a2-4b9c-957b-f62f5df02ead.png)
capacity filter was working fine.
![image](https://user-images.githubusercontent.com/109473873/226894485-50ba42b9-6e25-4aee-af63-13163ce18b1f.png)
and deploying successfully.
![image](https://user-images.githubusercontent.com/109473873/226896380-11a70581-c373-431f-b8bb-af495551555c.png)

And also tried to null the PK from polka and once logged into profile manager and applied the capacity filter, it  created a pk for the twin and deployed with no error.

But Tried with another twin having pk as '0x' and it didn't work, and no error was shown in the console.
![image](https://user-images.githubusercontent.com/109473873/226897986-528ed16b-6f0e-4e62-877b-71c38bbd4ca3.png)
![image](https://user-images.githubusercontent.com/109473873/226898328-e4ec67b0-dc12-4a3c-a924-05a142555125.png)

**Comment by A-Harby** [comment]
Closing as solved and opening new issue for old wrong pk https://github.com/threefoldtech/tfgrid-sdk-ts/issues/88
